src/Symfony/Bundle/FrameworkBundle/Routing/DelegatingLoader.php

```diff
@@ -11,11 +11,11 @@
 
 namespace Symfony\Bundle\FrameworkBundle\Routing;
 
+use Psr\Log\LoggerInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\ControllerNameParser;
 use Symfony\Component\Config\Exception\FileLoaderLoadException;
 use Symfony\Component\Config\Loader\DelegatingLoader as BaseDelegatingLoader;
 use Symfony\Component\Config\Loader\LoaderResolverInterface;
-use Psr\Log\LoggerInterface;
```

Comment:
Should be moved back, we don't reorder use statements to avoid merging conflict on other branches.